### PR TITLE
B/remove eval

### PR DESCRIPTION
**Pull request by mbartling**
Got rid of the evil eval that was causing problems all over with feed_dict.

### CODE REVIEW

File: utensor_cgen/parser/_pbparser_impl.py [hunk 1]
```diff
@@ -109,7 +110,8 @@ def _parse_graph_info(graph_def):
       if node.op in ["Const"]:
         for tensor_name, _, _ in output_tensor:
           tensor = graph.get_tensor_by_name(tensor_name)
-          output_content[tensor_name] = tensor.eval()
+          #output_content[tensor_name] = tensor.eval()
```
Comment (dboyliao) [inline utensor_cgen/parser/_pbparser_impl.py]: Just delete this line of code.
Good job.

File: utensor_cgen/parser/_pbparser_impl.py [hunk 2]
```diff
@@ -109,7 +110,8 @@ def _parse_graph_info(graph_def):
       if node.op in ["Const"]:
         for tensor_name, _, _ in output_tensor:
           tensor = graph.get_tensor_by_name(tensor_name)
```
Comment (dboyliao) [inline utensor_cgen/parser/_pbparser_impl.py]: And remove `tensor = graph.get_tensor_by_name(...)` too.
It's not needed anymore.

**Comment by mbartling** [comment]
@dboyliao fixed

File: utensor_cgen/parser/_pbparser_impl.py [hunk 3]
```diff
@@ -107,9 +108,7 @@ def _parse_graph_info(graph_def):
       output_content = {}
       op_attr = node.attr
       if node.op in ["Const"]:
-        for tensor_name, _, _ in output_tensor:
-          tensor = graph.get_tensor_by_name(tensor_name)
-          output_content[tensor_name] = tensor.eval()
+        output_content[tensor_name] = tensor_util.MakeNdarray(node.attr['value'].tensor)
```
Comment (dboyliao) [inline utensor_cgen/parser/_pbparser_impl.py]: You goes too far XD
You still need to loop over `output_tensor`.
Now the `tensor_name` is undefined. LOL

**Comment by dboyliao** [comment]
BTW, I think you should use `tensorflow.contrib.util.make_ndarray` instead of `MakeNdarray`.
Though they are just aliases now but `tensorflow.python.framework` is considered private.
(https://github.com/tensorflow/tensorflow/issues/798)
And they are not publicly well documented which means Google team might want to modify it in the future (I hope not....)
On the other hand, `tensorflow.contrib.util` is well(?) documented:
https://www.tensorflow.org/api_docs/python/tf/contrib/util

**Pull request merged**